docs/fields.asciidoc

```diff
@@ -43,6 +43,8 @@ Processor name.
 
 type: keyword
 
+{yes-icon} {ecs-ref}[ECS] field.
```

Comment:
The logic for discerning ECS fields in the docs is wrong, would you mind create an issue for that when you get a change?

Reply:
great catch!
https://github.com/elastic/apm-server/issues/4554